.changelog/26791.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+install: Updated license information displayed during post-install
```

Comment:
As this only impacts Enterprise installs, I think it's worth being explicit in the changelog using our standard Enterprise designation. This way, CE users won't see the entry and have to go digging to figure out whether it affects them.
```suggestion
install (Enterprise): Updated license information displayed during post-install
```